### DISCUSSION TITLE
可不可以做一些常用软件的开发基础模板

### DISCUSSION THREAD

**Discussion by hnothing2016** [netease-im/NIM_Duilib_Framework#329] [General]
像VS界面一样的界面模板，现在都是互联网软件的demo，没有这样的传统开发软件界面

![2021-05-28_105419](https://user-images.githubusercontent.com/20417103/119923357-20288400-bfa4-11eb-8eea-64f5963d5510.png)

**Comment by KnIfER** [comment]
…… 人家WFT，WPF种种开源的demo都有了，没必一模一样。有多样性、有特点就有价值。

**Comment by KnIfER** [comment]
https://user-images.githubusercontent.com/11593903/128175584-fdb10f01-44ae-4a40-bc49-36dd910bd2c4.mp4

你看我制作（魔改）的基于纯 win32 的子窗口系统多有特色啊，可以自定义顶、底面板的左右边界。

而且里面的每一个窗口都可以拿出来，可以最大化、最小化、贴边最大化(Aero Snapping)，通过双击标题栏或者拖拽又可以将之放回去，可以说兼具简洁与灵活。
